plugins/contests/ballot.js

```diff
@@ -0,0 +1,26 @@
+var ballot = {
+    author: 'pironic',
+    name: 'ballot',
+    description: '- Will output the number of ballots if called by a guest, but will enter you into the contest if you are not.',
+    config: {
+        permission: ['isListener']
+    },
+    onCall: function(request) {
+        var contestCore = require('./contestCore.js');
```

Comment:
If you wish to share parameter with all modules of your plugin, look for the init 'sharedObject'.
Eg:
```javascript
init: function(obj) {
   theObjectSharedBetweenAllModulesOfMyPlugin = obj.sharedObject;
},
```